assets/scss/base/_buttons.scss

```diff
@@ -77,6 +77,9 @@ button, .button {
   text-decoration: underline;
   color: $link-colour;
   border: none;
+  margin-top: em(10);
+  text-align: center;
+  box-sizing: border-box;
```

Comment:
Yes, as we discussed earlier, the implications of this need to be looked at across the services and cross-browser.